packages/adblocker-webextension/src/index.ts

```diff
@@ -539,19 +598,14 @@ export class WebExtensionBlocker extends FiltersEngine {
    * Deal with request cancellation (`{ cancel: true }`) and redirection (`{ redirectUrl: '...' }`).
    */
   public onBeforeRequest = (
-    browser: Browser,
+    _: Browser,
     details: OnBeforeRequestDetailsType,
   ): WebRequest.BlockingResponse => {
     const request = fromWebRequestDetails(details);
     if (this.config.guessRequestTypeFromUrl === true && request.type === 'other') {
       request.guessTypeOfRequest();
     }
 
-    if (request.isMainFrame()) {
-      this.performHTMLFiltering(browser, request);
```

Comment:
this cannot be simply removed - if any html filters are meant to be applied, they should prevent other filters from being run

Reply:
I take this back. We should prioritise blocking